inthe_am/taskmanager/models/taskstore.py

```diff
@@ -288,6 +293,10 @@ def _get_extra_safely(self, key, val):
                 self._is_numeric
             ),
             (
+                re.compile('^uda\.priority\.default$'),
```

Comment:
Ahh; for the moment, Inthe.AM is using TW 2.3 (but in the future, it'll be updated to use the latest version).  Could you add an additional exception for the old setting name, which I *think* was just `priority.default`?  If you can't or don't, no worries, I'll just take care of it when I get around to merging this.